src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
I don't think this hunk is correct.  You're saying here, that if WIN32 isn't defined, then unconditionally `#define HAVE_STPCPY`.  The problem here is that `HAVE_STPCPY` will only have been defined via autotools on UNIX, so I don't want this being declared defined if that's not true -- you're effectively clobbering its state here.